aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -196,9 +196,8 @@ AT_FORALL_SCALAR_TYPES_AND3(Bool, Half, BFloat16, DEFINE_CAST_OP)
 #undef DEFINE_CAST_OP
 
 Tensor empty_like(
```

Comment:
If we are collapsing, do we still need `native::empty_like(tensor)`?

Reply:
No, these can be deleted.